src/main/java/com/shorty/app/AdministrationService.java

```diff
@@ -0,0 +1,87 @@
+package com.shorty.app;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+
+import com.shorty.project.help.NameOfServer;
+import com.shorty.project.help.RandomString;
+
+@Service
+public class AdministrationService {
+	
+	@Autowired
+	AccountRepository accountRepository;
+	
+	@Autowired
+	UrlRepository urlRepository;
+
+	private BCryptPasswordEncoder bCryptPasswordEncoder = new BCryptPasswordEncoder(10);
+	
+	public String registerAccount(Account account) {
+		//If the username already exists
+		if(accountRepository.findByUsername(account.getUsername()) != null) {
+			return "{ success : false, description : 'Account ID already exists!' }"; 
```

Comment:
JSON response je hardkodiran. Da ne stavljam isti komentar, isto vrijedi za sve ostale metode u projektu koje trebaju vratiti JSON response.
Ako pogledaš listu stvari koje se treba dostaviti (iz word dokumenta s opisom zadatka - "4 Deliverables")
), tamo se spominje i "Swagger API specification".

Ispravan način je da si definirao API u swagger-u i na temelju swagger definicije genrirao clienta i korisito ga u projektu.
Hint: [Swagger](https://swagger.io/) , [Spring Boot & Swagger](https://www.springboottutorial.com/spring-boot-swagger-documentation-for-rest-services)